source/administration/object-management.rst

```diff
@@ -45,10 +40,25 @@ following:
          2020-01-02-MinIO-Advanced-Deployment-comments.json
          2020-01-04-MinIO-Interview.md
 
-MinIO supports multiple levels of nested directories and objects using a :term:`prefix` structure to support even the most dynamic object storage workloads.
+With the example structure, an administrator would create the ``/images``, ``/videos`` and ``articles`` buckets.
+Client applications would write objects to those buckets using the full "path" to that object, including all intermediate :term:`prefixes <prefix>`.
 
-MinIO itself does not limit the number of objects that any specific prefix can contain.
-However, hardware and network conditions may show performance impacts with large prefixes.
+MinIO supports multiple levels of nested directories and objects using prefixes to support even the most dynamic object storage workloads.
+Neither clients nor administrators would manually create the intermediate prefixes, as MinIO automatically infers them from the object name.
```

Comment:
> There is no need for clients or administrators to manually create prefixes, as MinIO automatically infers them from the object name.

Neither...nor constructions can be hard to follow, at least I find them difficult.

Reply:
Rewritten.